ocw_oer_export/create_csv.py

```diff
@@ -89,6 +107,20 @@ def get_cr_keywords(fm_ocw_keywords_mapping, list_of_topics_objs, course_url):
     return "|".join(topic["name"] for topic in list_of_topics_objs)
 
 
+def get_cr_create_date(semester, year):
```

Comment:
**Suggestion:** Some OCW courses do not have a year. In that case, the current code will return `None-01-01`. 

Instead, add `if not year: return ""`.

**However:** A bug is currently preventing MIT Open from including these courses. See https://github.com/mitodl/mit-open/issues/441. So a little hard to test this change on real data right now.